lib/travis/build/env/builtin.rb

```diff
@@ -53,6 +53,7 @@ def env_vars
               TRAVIS_PULL_REQUEST_BRANCH: job[:pull_request_head_branch],
               TRAVIS_PULL_REQUEST_SHA: job[:pull_request_head_sha],
               TRAVIS_PULL_REQUEST_SLUG: job[:pull_request_head_slug],
+              TRAVIS_JOB_NAME:        job[:name]
```

Comment:
It's a small thing, but adding a trailing `,` helps to minimize the next PR.

Also, while there is no order in `TRAVIS_*` env vars, it seems a good idea to keep this one close to other `TRAVIS_JOB_*` env vars.